platform_versioned_docs/version-24.1/data_studios/custom-envs.mdx

```diff
@@ -0,0 +1,143 @@
+---
+title: "Custom environments"
+description: "Custom environments for Data Studios"
+date: "1 Oct 2024"
+tags: [data, studios]
+---
+
+In addition to the Seqera-provided container template images, you can provide your own custom container environments, either by augmenting the Seqera-provided images with a list of Conda packages that you provide or by providing your own base container template image.
+
+## Conda packages
+
+### Prerequisites
+
+- Wave must be configured. For more information, see [Wave containers][wave].
+
+### Conda package syntax {#conda-package-syntax}
+
+When you add a new data studio, you can customize its configuration to install a list of Conda packages to the template. The supported schema is identical to that used by the Conda `environment.yml` environment file. For more information on the Conda environment file, see [Creating an environment file manually][env-manually].
+
+```yaml title="Example environment.yml file"
+name: example-env
+channels:
+  - defaults
+  - conda-forge
+dependencies:
+  - python=3.7
+  - numpy
+  - pandas
+  - scikit-learn
+  - pip:
+    - matplotlib
+    - seaborn
+    - jupyter
+```
+
+To create a data studio with custom Conda packages, see [Add a data studio][add-ds].
+
+## Custom container template image {#custom-containers}
+
+For advanced use cases, you can build your own container template image.
+
+:::note
+Amazon Elastic Container Registry (ECR) is the only supported private container registry.
+:::
+
+### Prerequisites
+
+- Access to a container image repository, either a public container registry or in a private Amazon ECR repository
```

Comment:
I'm not sure if users need to have credentials added to the workspace in order to access the container registry

```suggestion
- Access to a container image repository, either a public container registry or a private Amazon ECR repository
```